likecoin/js/admin-settings/src/components/MainSettingTable.js

```diff
@@ -206,30 +218,17 @@ function MainSettingTable(props) {
             editable={true}
             isMainSettingPage={true}
             showChangeButton={true}
-            showDisconnectButton={false}
-          />
-        )}
-        </tbody>
-        <Section
-          title={__('Site LikeCoin button display setting', 'likecoin')}
-        />
-        <tbody>
-          <DropDown
-            selected={displayOptionSelected}
-            handleSelect={setDisplayOptionSelected}
-            title={__('Display option', 'likecoin')}
-            selectRef={displayOptionRef}
-            options={pluginSettingOptions}
+            showDisconnectButton={true}
           />
           <CheckBox
-            checked={perPostOptionEnabled}
-            handleCheck={setPerPostOptionEnabled}
-            title={__('Allow per Post option', 'likecoin')}
+            checked={siteLikerIdEnabled}
+            handleCheck={setSiteLikerIdEnabled}
+            title={__('Override all Liker ID with site default', 'likecoin')}
             details={__(
-              'Allow editors to customize display setting per post',
+              'Override all LikeCoin button with site default Liker ID',
```

Comment:
```suggestion
              'Override all LikeCoin widgets with site default Liker ID',
```

Reply:
I think it is still called `LikeCoin button` everywhere in this release, maybe we will handling naming in a later PR for  production release